choreolib/src/main/native/cpp/choreo/lib/ChoreoTrajectoryState.cpp

```diff
@@ -49,14 +53,24 @@ frc::ChassisSpeeds ChoreoTrajectoryState::GetChassisSpeeds() const {
 ChoreoTrajectoryState ChoreoTrajectoryState::Interpolate(
     const ChoreoTrajectoryState& endValue, double i) const {
   frc::Pose2d interpPose = frc::Interpolate(GetPose(), endValue.GetPose(), i);
+
+  // LERP all force values in both directions.
+  std::array<units::newton_t, 4> lerpFX, lerpFY;
+  for (int idx = 0; idx < 4; ++idx) {
+    lerpFX[idx] = wpi::Lerp(moduleForcesX[idx], endValue.moduleForcesX[idx], i);
+    lerpFY[idx] = wpi::Lerp(moduleForcesY[idx], endValue.moduleForcesY[idx], i);
+  }
+
   return ChoreoTrajectoryState{
       wpi::Lerp(timestamp, endValue.timestamp, i),
       interpPose.X(),
       interpPose.Y(),
       interpPose.Rotation().Radians(),
       wpi::Lerp(velocityX, endValue.velocityX, i),
       wpi::Lerp(velocityY, endValue.velocityY, i),
-      wpi::Lerp(angularVelocity, endValue.angularVelocity, i)};
+      wpi::Lerp(angularVelocity, endValue.angularVelocity, i),
+      lerpFX,
+      lerpFY};
 }
 
 std::array<double, 7> ChoreoTrajectoryState::AsArray() const {
```

Comment:
Should this include module data now? Leaning towards no.

Reply:
Idk what this function is even for, cuz we already support serialization a different way.

Reply:
I'll just leave it be then.